tiatoolbox/tools/stainaugment.py

```diff
@@ -101,13 +102,14 @@ class StainAugmentor(ImageOnlyTransform):
     def __init__(
         self,
         method: str = "vahadane",
-        stain_matrix: np.ndarray = None,
+        stain_matrix: Optional[np.ndarray] = None,
         sigma1: float = 0.4,
         sigma2: float = 0.2,
         augment_background: bool = False,
         always_apply=False,
         p=0.5,
-    ) -> np.ndarray:
+    ) -> None:
+        """Initializes :class:`StainAugmentor`."""
```

Comment:
```suggestion
        """Initialize :class:`StainAugmentor`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090